docs/api/options.md

```diff
@@ -60,6 +60,7 @@
   - `subscription.onConnect`: `Function` A function which can be used to validate the `connection_init` payload. If defined it should return a truthy value to authorize the connection. If it returns an object the subscription context will be extended with the returned object.
   - `subscription.onDisconnect`: `Function` A function which is called with the subscription context of the connection after the connection gets disconnected.
   - `subscription.keepAlive`: `Integer` Optional interval in ms to send the `GQL_CONNECTION_KEEP_ALIVE` message.
+  - `subscription.fullWsTransport`: `Boolean` Enable sending every operation via WS.
```

Comment:
I think some more heavy-handed changes to our docs are needed. I think we should flip the config and  have a `websocket` top level option.

Reply:
Ok, make sense! But do we want to assume a breaking change? How would you structure the new config?

Reply:
I think this is the only point left so far, but I'm not sure about how to introduce those changes into the config/docs right now.

Reply:
Let's not do a breaking change but support both the new and old options.

Reply:
Thinking about adding a `websocket` option on the config top-level, not sure it would be better... if you mean to add it to group related configs, I think that  `subscription` is actually a better choice, taking in mind that subscriptions could also be implemented with SSE (there's already an open discussion about that), and also that doesn't make sense to setup any websocket if you don't use subscriptions... 🤔

Reply:
let's go with this option and we could refactor later